samples/sensor/6dof_motion_drdy/README.rst

```diff
@@ -0,0 +1,49 @@
+.. _6dof_motion_drdy:
+
+6DOF Motion Dataready:
+############################################
```

Comment:
wrong formatting

Reply:
I updated both README for 6dof_motion_drdy and tdk_apex samples.